R/sidebar.R

```diff
@@ -185,39 +185,50 @@ sidebarServer <- function(id, uploaded_pkgs) {
                No - Exits from window and removes the text in comment box."
           ),
           footer = tagList(
-            actionButton("submit_overall_comment_yes", "Yes", class = "btn-success"),
-            actionButton("submit_overall_comment_edit", "Edit", class = "btn-secondary"),
-            actionButton("submit_overall_comment_no", "No", class = "btn-unsuccess")
+            actionButton(NS(id, "submit_overall_comment_yes"), "Yes", class = "btn-success"),
+            actionButton(NS(id, "submit_overall_comment_edit"), "Edit", class = "btn-secondary"),
+            actionButton(NS(id, "submit_overall_comment_no"), "No", class = "btn-unsuccess")
```

Comment:
Oh looky here! I've never actually used `NS()` on each UI id, but I see that's what Hadley is doing in Mastering Shiny. Previously, we always started any UI script with: 

`ns <- NS(id)`

and then called `ns()` in it's place, so line 188 above would start out like...

`actionButton(ns("submit_overall_comment_yes"), "Yes", class = "btn-success"),`

Going forward, do we want to use little ns or big ns?

Reply:
As long as we're consistent, @AARON-CLARK 
Yes, I'm OK with using little ns.

Reply:
Actually, when I do
![image](https://user-images.githubusercontent.com/75215958/156222879-bfa6663b-ce7e-44c5-a6a9-c34dd9f8d5fc.png)
I find all the other functions are using big NS()
So maybe we should just leave it be.

Reply:
Let's leave it as-is then!